test/validators.js

```diff
@@ -6719,6 +6719,14 @@ describe('Validators', () => {
           '22040-020',
           '39400-152',
         ],
+        invalid: [
```

Comment:
I also made sure that I added tests for invalid cases. Just for safety. 

Cc. @profnandaa.

Reply:
Looks good!